clients/privacy-center/cypress/e2e/consent-banner-gpp.cy.ts

```diff
@@ -554,4 +554,202 @@ describe("Fides-js GPP extension", () => {
       });
     });
   });
+
+  describe("with TCF disabled and GPP enabled, with experience region outside of GPP", () => {
```

Comment:
This feels like way too much copy-pasta'ing, sorry... can we avoid duplicating _all_ these tests here?

Reply:
Couple suggestions:
1. You could extract the contents of the `beforeEach` block into a `visitDemoWithGPP` helper that provides an option to override just the `experience` part; see the `visitDemoWithI18n` helper I did for i18n tests here: https://github.com/ethyca/fides/blob/79e8c845953ae683647a63874328692ff2a64f63/clients/privacy-center/cypress/e2e/consent-i18n.cy.ts#L355. You could then organize the tests to be something like:
  - describe("with TCF disabled and GPP enabled")
    - describe("when visiting from a state with an applicable section")
      - beforeEach -> setup GPP but from an applicable state
      - (...all existing tests)
    - describe("when visiting from a state that does not have an applicable section")
      - beforeEach -> setup GPP but from outside (e.g. NC)
      - (...just one or two tests for this case without duplicating everything)
2. You could instead refactor this to be a parametrized test runner that sets up cases and expectations and then does the `testCases.forEach(...)` magic to reuse the same describe/it blocks for all the cases

In this case option #1 feels reasonable, it'd allow you to avoid _too much_ duplication and just DRY up some of the shared logic.